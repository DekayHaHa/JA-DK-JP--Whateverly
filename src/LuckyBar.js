import React from 'react';
import './LuckyBar.css';

export default function LuckyBar(props) {
  return(
    <div className="lucky-bar search-section">
      <button className="button lucky random-all" onClick={props.filter}>Random from All</button>
      <button className="button lucky random-filter" onClick={props.randomFilter}>Random from Filters</button>
    </div>
  )
}