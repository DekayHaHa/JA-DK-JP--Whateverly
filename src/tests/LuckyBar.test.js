import React from 'react';
import ReactDOM from 'react-dom';
import LuckyBar from '../LuckyBar';
import { shallow } from 'enzyme';

describe('LuckyBar', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <LuckyBar />
    );
  });

  it('should match snapshop', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should have default state', () => {
    expect(wrapper.state()).toEqual({
      active: false
    });
  })
})