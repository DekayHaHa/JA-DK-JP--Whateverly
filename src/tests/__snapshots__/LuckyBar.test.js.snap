// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LuckyBar should match snapshop 1`] = `
<div
  className="lucky-bar search-section"
>
  <button
    className="button lucky random-all"
  >
    Random from All
  </button>
  <button
    className="button lucky random-filter"
  >
    Random from Filters
  </button>
</div>
`;
